client/makelunch.less

```diff
@@ -175,7 +175,18 @@ body.home .card {
       .btn { display: block; }
     }
   }
-  &:hover, &:focus {
+  @media (min-width: 768px) {
```

Comment:
Although we (I) did inline media queries a lot on School of Wok, I think we've all decided it's not great in the long run - have them all together in a block at the bottom of the file.